packages/formation-react/src/components/Sidebar/SidebarBlock.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SidebarBlock = props => {
+  let headingContent = (
+    <h2 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary sidebar-heading">
+      {props.heading}
+    </h2>
+  );
+
+  if (props.headingLevel === 3) {
+    headingContent = (
+      <h3 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary sidebar-heading">
+        {props.heading}
+      </h3>
+    );
+  }
+
+  if (props.headingLevel === 4) {
+    headingContent = (
+      <h4 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary sidebar-heading">
+        {props.heading}
+      </h4>
+    );
+  }
```

Comment:
I think this `headingLevel` logic could be condensed a bit. The simplest might just be to store the value of the `className` in a variable then maybe use a switch statement for the `props.headingLevel`.

Reply:
Yep, I agree. Moved it into a switch statement